src/app/components/drawer/SavedSearches/SavedSearchActions.js

```diff
@@ -210,19 +224,19 @@ const ProjectActions = ({
         cancelLabel={<FormattedMessage defaultMessage="Cancel" description="Generic label for a button or link for a user to press when they wish to abort an in-progress operation" id="global.cancel" />}
         isSaving={saving}
         open={showEditDialog}
-        proceedDisabled={!newTitle && !object.title}
+        proceedDisabled={!newTitle && !savedSearch.title}
         proceedLabel={
           <FormattedMessage
             defaultMessage="Rename list"
             description="'Rename' here is an infinitive verb"
-            id="projectsComponent.renameType"
+            id="savedSearchActions.renameType"
```

Comment:
Please let's keep existring string ids as they were to avoid unnecessary re-translation work and costs.

Reply:
Done